src/components/FilterButton/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { Link } from 'react-router-dom'
+import { Popover } from 'antd'
+import { ReactComponent as FilterIcon } from '../../assets/filter_icon.svg'
+import { ReactComponent as SelectedCheckIcon } from '../../assets/selected_check_icon.svg'
+import { useSearchParams } from '../../hooks'
+import styles from './styles.module.scss'
+
+export function FilterButton({
+  filteredList,
+  isMobile,
+}: {
+  filteredList: Record<'title' | 'value' | 'to', string>[]
+  isMobile?: boolean
+}) {
+  const { type } = useSearchParams('type')
+
+  return (
+    <Popover
+      className={styles.container}
+      placement="bottomRight"
+      trigger={isMobile ? 'click' : 'hover'}
+      overlayClassName={styles.antPopover}
+      content={
+        <div className={styles.filterItems}>
+          {filteredList.map(f => (
+            <Link key={f.value} to={`${f.to}?type=${f.value}`} data-is-active={f.value === type}>
```

Comment:
It's better to rename the Fitler as TypeFilter if the keyword is `type`, which is hardcoded in the URL.

Reply:
f39491f0